assets/runtime/config/gitlabhq/cable.yml

```diff
@@ -0,0 +1,14 @@
+# This is a template taken from here:
+# https://gitlab.com/gitlab-org/gitlab/-/blob/master/config/cable.yml.example
+development:
```

Comment:
If the definition of the Redis connection for the environments 'development' and 'test' is not needed, then I would suggest to either use other "default values" (like
`url: redis://:fakepass@fake.redis.actioncable.com:1234` and `channel_prefix: gitlab_fake_development`) or not be defined at all.